tests/platform_tests/link_flap/test_cont_link_flap.py

```diff
@@ -208,26 +207,25 @@ def test_cont_link_flap(self, request, duthosts, nbrhosts, enum_rand_one_per_hws
             logging.info("Orchagent PID {0} CPU Util at end: {1}".format(pid, util))
 
         # Record Redis Memory at end
-        end_time_redis_memory = duthost.shell(
-            r"redis-cli info memory | grep used_memory_human | sed -e 's/.*:\(.*\)M/\1/'")["stdout"]
-        logging.info("Redis Memory at start: %s M", start_time_redis_memory)
-        logging.info("Redis Memory at end: %s M", end_time_redis_memory)
+        end_time_redis_memory = get_avg_redis_mem_usage(duthost, 5, 5)
+        logging.info("Redis Memory at start: %f M", start_time_redis_memory)
+        logging.info("Redis Memory at end: %f M", end_time_redis_memory)
 
         # Calculate diff in Redis memory
-        incr_redis_memory = float(end_time_redis_memory) - float(start_time_redis_memory)
-        logging.info("Redis absolute  difference: %d", incr_redis_memory)
+        incr_redis_memory = end_time_redis_memory - start_time_redis_memory
+        logging.info("Redis absolute difference: %f", incr_redis_memory)
```

Comment:
nit: this isn't an absolute difference - "Redis memory difference: %f MB"

Reply:
Resolved